app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    if len(command_parts) != 3:
+        return
+
+    action, source_file, destination_file = command_parts
+    if action != "cp" or source_file == destination_file:
+        raise
```

Comment:
In case you use `raise`, add any exception with meaningful text, do not leave it as it is.